Documentation/capabilities-guide.md

```diff
@@ -0,0 +1,235 @@
+# Capabilities Isolators Guide
+
+This document is a walk-through guide describing how to use rkt isolators for
+[Linux Capabilities](https://lwn.net/Kernel/Index/#Capabilities).
+
+* [About Linux Capabilities](#about-linux-capabilities)
+* [Default Capabilities](#default-capabilities)
+* [Capability Isolators](#capability-isolators)
+* [Usage Example](#usage-example)
+* [Recommendations](#recommendations)
+
+## About Linux Capabilities
+
+Linux capabilities are meant to be a modern evolution of traditional UNIX
+permissions checks.
+The goal is to split the permissions granted to privileged processes into a set
+of capabilities (eg. `CAP_SYS_CHROOT` to perform a `chroot(2)`), which can be
+separately handled and assigned to single threads.
+
+Processes can gain specific capabilities by either being run by superuser, or by
+having the setuid/setgid bits or specific file-capabilities set on their
+executable file.
+Once running, each process has a bounding set of capabilities which it can
+enable and use; such process cannot get further capabilities outside of this set
+(except by using one of methods mentioned above).
```

Comment:
My understanding was that if a cap is not in the capability bounding set, the process cannot get it back, even by running a setuid binary.

`no_new_privs` is for forbidding a process to get caps in its effective set, even if it still has the caps in its bounding set.